app/forms/steps/income/business_financials_form.rb

```diff
@@ -1,22 +1,30 @@
 module Steps
   module Income
     class BusinessFinancialsForm < Steps::BaseFormObject
-      attribute :turnover, :amount_and_frequency
-      attribute :drawings, :amount_and_frequency
-      attribute :profit, :amount_and_frequency
+      attribute :turnover, :amount_and_frequency, default: {}
+      attribute :drawings, :amount_and_frequency, default: {}
+      attribute :profit, :amount_and_frequency, default: {}
 
-      validates :turnover, :drawings, :profit, payment: true
+      delegate(:amount, :amount=, :frequency, :frequency=, to: :turnover, prefix: true)
+      delegate(:amount, :amount=, :frequency, :frequency=, to: :drawings, prefix: true)
+      delegate(:amount, :amount=, :frequency, :frequency=, to: :profit, prefix: true)
+
```

Comment:
Form users #turnover_amount instead of #turnover.amount etc., to allow default GOVUK formbuilder and rails error messaging.